blobfuse/src/BlobfuseGcCache.cpp

```diff
@@ -184,7 +188,7 @@ void gc_cache::run_gc_cache()
         else
         {
             // no file was timed out - let's wait a second
-            usleep(1000);
```

Comment:
is this related to not listing on mount?

Reply:
this was one ms sleep was was converted to 1 sec sleep that time. Now we need to make this configurable.